tools/cldr/gencurrencies.js

```diff
@@ -111,7 +111,7 @@ if (process.argv.length < 2) {
 }
 
 if (process.argv.length > 2) {
-    ilibDir = process.argv[2];
```

Comment:
This change might give confusion. 
Is for referencing ilib location while generating data. It's not for output location.

When running gencurencies.js, the ilib locale data path should be put as a parameter mandatory. because if the cldr doesn't have proper data, we've decided to maintain current ilib data.
Usage: gencurrency [iLib locale Dir] [toDir]

Reply:
put it back the way it was, but added a declaration for the variable that node complained about